core/templates/components/concept-card/concept-card.directive.ts

```diff
@@ -46,17 +46,18 @@ angular.module('oppia').directive('conceptCard', [
           };
 
           ctrl.showMoreWorkedExamples = function() {
+            ctrl.answerIsShown = false;
             ctrl.numberOfWorkedExamplesShown++;
           };
 
           ctrl.$onInit = function() {
             ctrl.conceptCards = [];
             ctrl.currentConceptCard = null;
-            ctrl.numberOfWorkedExamplesShown = 0;
+            ctrl.numberOfWorkedExamplesShown = 1;
```

Comment:
Make sure to correctly handle the case where no worked examples have been entered for a concept card yet.

Reply:
I didn't completely understand why we are setting the init value to be 1?

Reply:
Since we aren't showing the explanation, but just the question, I thought it might be good to show an example question by default so that the learner can see a question as they are reading the skill and then check its answer and view other examples, if needed. Does that sound fine?

Reply:
Done

Reply:
That makes sense but yeah there may be a case where we have no examples, just wanted to make sure the UI and experience looks okay for that too!